src/data.js

```diff
@@ -0,0 +1,324 @@
+const movieData =  [
+    {
```

Comment:
Rather than try to access the object value from movie data, we just removes the key-value pair, and made movieData an Array